lib/passenger.rb

```diff
@@ -16,6 +16,25 @@ def add_trip(trip)
       @trips << trip
     end
 
+    def net_expenditures
+      return nil if @trips.empty?
+      total = 0
+      @trips.each do |trip| #why didn't this work with .reduce?
+        if trip.cost.nil?
+          puts "There is a trip in progress that will not be included in total."
```

Comment:
This puts statement definitely helps make sure everything is going right, but it's general best practice not to include puts statements in a method that is fundamentally there to complete a calculation.